instrumentation-genai/opentelemetry-instrumentation-openai-v2/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Support for `AsyncOpenAI/AsyncCompletions`
```

Comment:
Nit: Please use the similar format for CHANGELOG as the other entries

Reply:
@alizenhom I think this means put the PR here in parens like those below.